atlasdb-autobatch/src/main/metrics/autobatchers.yml

```diff
@@ -0,0 +1,54 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.autobatch'
+
+namespaces:
+  autobatchOverhead:
+    docs: Metrics on how much time was spent waiting for tasks to enqueue and running tasks
+    tags:
+      - operationType
+    metrics:
+      waitTimeMillis:
+        docs: Time spent waiting
+        type: histogram
+      waitTimeMillisP1:
+        docs: Time spent waiting (p1 percentile only)
+        type: gauge
+      waitTimeMillisP5:
+        docs: Time spent waiting (p5 percentile only)
+        type: gauge
+      waitTimeMillisP50:
```

Comment:
This does not show. The internal filters are based on suffix (not matching the histogram metric name) probably. This is why P1/P5/P999 show up (they are not explicitly filtered out, or similar).

Most likely will change the name of this to waitTimeMillisMedian (and similar to the corresponding ones)